cypress/features/regression/splitButtonClassic.feature

```diff
@@ -5,64 +5,61 @@ Feature: Split Button component
     Given I open "Split Button" component page classic
 
   @positive
-  Scenario Outline: I select as to <as>
-    When I select as to "<as>"
-    Then Button background color is "<color>"
-    Examples:
-      | as        | color            |
-      | primary   | rgb(37, 91, 199) |
-      | secondary | rgba(0, 0, 0, 0) |
+  Scenario: I expand Split Button component classic page
+    When I hover mouse onto icon
+    Then Split Button is expanded
 
   @positive
-  Scenario Outline: I set data-element to <data-element>
-    When I set data-element to "<data-element>"
-    Then data-element "<data-element>" is present
+  Scenario Outline: Verify color palette for Split Button component without focus
+    # When I open "Split Button" component page
+    Then Split Button first element has proper background-color "<color>" and border "<border-color>" color and has border-width 1 px
+      And Split Button second element has proper background-color "<color>" and border "<border-color>" color and has border-width 1 px
     Examples:
-      | data-element            |
-      | Sample text             |
-      | 1234567890              |
-      | áéíóú¿¡üñ               |
-      | !@#$%^*()_+-=~[];:.,?{} |
-      | ÄÖÜßäöüß                |
+      | color            | border-color     |
+      | rgba(0, 0, 0, 0) | rgb(37, 91, 199) |
 
-  @positive
-  Scenario Outline: I set data-role to <data-role>
-    When I set data-role to "<data-role>"
-    Then data-role "<data-role>" is present
+  @ignore
```

Comment:
Please, add a comment why this TC has 'ignore' tag

Reply:
added